ext/tags.go

```diff
@@ -88,6 +88,18 @@ var (
 
 	// Error indicates that operation represented by the span resulted in an error.
 	Error = boolTagName("error")
+
+	//////////////////////////////////////////////////////////////////////
+	// PerfEvent Tag
+	//////////////////////////////////////////////////////////////////////
+
+	// PerfEvent records the list of events given in a comma separated string (e.g.
+	// "cpu-cycles,cache-misses,branch-misses") for a span.
+	// The tracer is expected to start the events in PerfEvent using the "perfevents"
+	// package (https://github.com/opentracing-contrib/perfevents/go) . On span
+	// Finish(), the events must be closed.
+
+	PerfEvent = stringTagName("perfevents")
```

Comment:
@yurishkuro I think that makes plenty of sense. I would like to avoid things in `opentracing-contrib` that require action by `Tracer` implementors... it just seems unrealistic / unsustainable.

More generally I see a benefit (due to intra-process propagation) from separating OT API and SPI interfaces. It looks good on paper, but I think it's tricky to do right with sampling and noop impls taken into account.

Sorry for this tangent, @hkshaw1990 ... it's for a good cause :)